charts/rancher-logging/0.0.2/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+description: Rancher logging helm chart to support logging function in rancher
+icon: https://raw.githubusercontent.com/fluent/fluentd-docs/master/public/logo/Fluentd_square.png
+name: rancher-logging
+version: 0.1.1
```

Comment:
The version here is not matched with your chart’s path(0.0.2). Should keep them in sync